### PR TITLE
chore: reduce default instance size everywhere

### PR DESCRIPTION
**Pull request by triclambert**

### CODE REVIEW

File: packages/cli/src/schemas.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ const CONTROLLER_IMAGE_DEFAULT = `opstrace/controller:${BUILD_INFO.VERSION_STRIN
 // `us-west-2a` encodes availability zone "a" within that region.
 export const infraConfigSchemaAWS = yup
   .object({
-    instance_type: yup.string().default("t3.2xlarge"),
+    instance_type: yup.string().default("t3.xlarge"),
```
Comment (jgehrcke) [inline packages/cli/src/schemas.ts:31]: If we make this change a documentation upadte has to be part of this :)

https://github.com/opstrace/opstrace/blob/3f405cd4baa709c5d624d8966b8e2820b28ea37f/docs/references/cluster-configuration.md#aws

Comment (jgehrcke) [reply packages/cli/src/schemas.ts:31]: edit: oh, the default values are not yet explicitly documented. OK.

**Comment by jgehrcke** [comment]
context: https://github.com/opstrace/opstrace/issues/262

**Comment by sreis** [comment]
Pushed a commit to update the docs and stopped the build. Although not explicit, might as well keep the config example in sync with the defaults.

**Comment by jgehrcke** [comment]
> Although not explicit, might as well keep the config example in sync with the defaults.

yes, further down the line we should document (as with `gcp`) that what's shown _are_ the defaults, and even further down the line this kind of thing should be automated :)

**Pull request merged**